src/Pages/SignUp/Components/SignUpList/SignUpList.js

```diff
@@ -0,0 +1,71 @@
+import React, { Component } from 'react';
+
+class SignUpList extends Component {
+  render() {
+    const {
+      index,
+      type,
+      title,
+      name,
+      placeholder,
+      validator,
+      stateName,
+    } = this.props;
+    return (
+      <li className="table">
+        <div className="tableName">
+          <span className="tableIcon">■</span>
+          <span>{this.props.title}</span>
```

Comment:
```suggestion
          <span>{title}</span>
```
위에서 구조분해 할당 해놓으시고 사용 안하신거 같습니다!